src/main/java/org/alliancegenome/curation_api/services/validation/dto/fms/GeneExpressionAnnotationFmsDTOValidator.java

```diff
@@ -48,6 +48,7 @@ public class GeneExpressionAnnotationFmsDTOValidator {
 	public GeneExpressionAnnotation validateAnnotation(GeneExpressionFmsDTO geneExpressionFmsDTO, BackendBulkDataProvider dataProvider, Map<String, Set<String>> experiments) throws ValidationException {
 		ObjectResponse<GeneExpressionAnnotation> response = new ObjectResponse<>();
 		GeneExpressionAnnotation geneExpressionAnnotation = new GeneExpressionAnnotation();
+		geneExpressionAnnotation.setExpressionPattern(new ExpressionPattern());
```

Comment:
Do we need lines 67-69 as well as this?

Reply:
If you remove this line and move lines 67-69 to after the `if (singleReferenceResponse.hasErrors())` if/else clause then you can get rid of this duplication.

Reply:
Done